go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/ethstorage/go-ethstorage
 
-go 1.20
+go 1.21
```

Comment:
We may need to update the guide [here](https://github.com/ethstorage/es-node/blob/main/GUIDE.md#system-environment)

Reply:
Also adding @syntrust here to check if there are other places to update